### DISCUSSION TITLE
독서 메모법 관련 유투브 영상 추천

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#207] [General]
[한 번만 읽어도 책 내용 다 기억하는 기적의 독서 메모 법](https://www.youtube.com/watch?v=fxdPHw8vhog)

이라는 유투브 영상을 보았는데요 제목이 좀 어그로틱하긴한데, 영상에서 말하는 내용들은 충분히 실천해봄직한 것들을 많이 알려주셔서 추천드려봅니다~ 😄 

추가로, 영상을 보고 의견을 여기에서 나눠봐도 좋을 것 같네요 👍🏻

**Comment by TaeHyoungKwon** [comment]
TMI,

위 영상을 보고 저는 오늘 부터 

1. `만능 노트` 를 써보려고 합니다
2. `순간의 생각`을 적극 활용해보려고 합니다

이전부터 메모들이 여기저기 흩어져서 정리가 안되는게 고민이였는데, `만능노트` 와 `정리노트` 를 나누어서 관리하는 것이 이 문제를 해결해 줄 수 있을 것 같은 기대가 듭니다 ㅎㅎ

**Reply by jongfeel** [reply]
저도 영상 보고 떠오른 것, 그리고 저도 해보면 좋겠다고 생각한 것을 적어 보면

- 만능 노트: 사람들과 얘기하면서 얻은 좋은 아이디어 적기

입니다.

사실 나중에 하면 좋겠다고 생각한 것들은 google keep이나 회사 github 어딘가에 적어 놓기는 하는데
한 곳에 모여있지 않다 보니 몇 달 이후에 발견하게 되는 아쉬움이 있다는 것도 영상을 보고 알게 되었습니다.

저도 태형님처럼 이곳 저곳 적어 두기는 하는데 정리가 잘 안되고 있었던 상태인 것이죠 ㅎㅎㅎ

그런데!
저는 펜과 종이에 쓰는 걸 선호하지 않아서
가지고 있는 iPad mini 6와 애플 펜슬을 좀 활용해 보려 합니다.
어떤 앱을 쓸지는 이제 고민해 봐야 겠습니다 ㅎㅎ

**Reply by Joseph-Cha** [reply]
와 저도 영상 보면서 이건 진짜 해보면 너무 좋겠단 생각이 계속 뿜뿜하네요.
저는 우선 순간의 생각이랑 생각의 이음을 적극적으로 연습을 해보려구요.
책만 보고 있으면 어느 순간 맥락을 놓치는 경우가 많이 있는데 이게 그냥 책을 읽으려고만 하기 때문인 것 같아요. 

좋은 영상 공유해주셔서 너무 감사합니다.
덕분에 좋은 인사이트를 얻고 가요 ㅎㅎ
